CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ All notable changes to this project will be documented in this file.
 
 - Enhance the PR template. ([#4881](https://github.com/wazuh/wazuh-qa/pull/4881)) \- (Framework)
 
+### Fixed
+
+- Fixed agent_simulator response for active-response configuration commands. ([#4895](https://github.com/wazuh/wazuh-qa/pull/4895)) \- (Framework)
```

Comment:
Include this section below `4.8.2`

Reply:
Changed in commit 57dd3a9e6e087d489a356d53f106b1c99bb814d1. Thanks!